docs/content/docs/user-docs/cross-account-resource-management.md

```diff
@@ -83,6 +83,212 @@ spec:
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:ListRoles",
+                "sts:AssumeRole"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOF
+```
+
+Then use the file to create a policy in account 000000000000:
+```bash
+aws --profile 000000000000 iam create-policy \
+  --policy-name ack-can-assume \
+  --policy-document file://user-policy.json
+```
+
+You should see a new policy ARN in the output of the previous command. Copy it and use it to attach the policy to the `ack-elasticache-service-controller` user:
+```bash
+aws --profile 000000000000 iam attach-user-policy \
+  --user-name ack-elasticache-service-controller \
+  --policy-arn "arn:aws:iam::000000000000:policy/ack-can-assume"
+```
+
+Now you must create a trust policy in any secondary accounts, i.e. account 111111111111, which allows `ack-elasticache-service-controller` to assume their role. Create the following file with the `$USER_ARN` you saved in step 1:
+```bash
+cat > trust-policy.json <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Principal": {
+            "AWS": "$USER_ARN"
+        },
+        "Action": "sts:AssumeRole"
+    }
+}
+EOF
+```
+
+Use this file to create the role in the secondary account:
+```bash
+aws --profile 111111111111 iam create-role \
+  --role-name ack-manage-elasticache \
+  --assume-role-policy-document file://trust-policy.json
+```
+
+Save the role ARN from this output for use in step 4, i.e.:
+```bash
+ROLE_ARN="arn:aws:iam::111111111111:role/ack-manage-elasticache"
+```
+
+
+Give the role permissions using a policy (select the appropriate permission ARN for your use case):
+```bash
+aws --profile 111111111111 iam attach-role-policy \
+    --role-name ack-manage-elasticache \
+    --policy-arn 'arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess'
+```
+
+You can verify that the policy applied properly with:
+```bash
+aws --profile 111111111111 iam list-attached-role-policies \
+    --role-name ack-manage-elasticache
+```
+
+### Step 3: Create the default ACK namespace
+
+Create the namespace for any ACK controllers you might install. The controllers as they are packaged in OperatorHub and OLM expect the namespace to be `ack-system`.
+```bash
+oc new-project ack-system
+```
+
+### Step 4: Configure the ACK controller to change accounts based on namespace
+
+To bind AWS accounts to a specific namespace you will have to annotate the namespace with an AWS account ID. To do so in an existing namespace:
+```bash
+oc annotate namespace production services.k8s.aws/owner-account-id=111111111111
+```
+
+You must also create a `ConfigMap` named `ack-role-account-map` in the `ack-system` namespace where the controller is running, using the `$ROLE_ARN` you saved earlier in step 2:
+```bash
+oc create configmap \
+  --namespace ack-system \
+  --from-literal=111111111111=$ROLE_ARN \
+  ack-role-account-map
+```
```

Comment:
`oc` is a super set of `kubectl`, some `cmds` are inherited and some are not. For OpenShift specific cluster it's better to use `oc` `cmds` and for us to have that in the documentation for clarity.

Reply:
Looking through these comments though, @jaypipes is right that a lot of this is actually just a more verbose version of the instructions above for kubectl stuff. I think we could probably just add a note block somewhere near the top that you should use `oc` commands in place of `kubectl` if you're on OpenShfit. I didn't realize that even the namespace and the assume role config map are named the same. I can simplify a lot of this.

Reply:
And also on that note, `oc` commands I think have almost 1 to 1 feature parity with `kubectl` commands, it's just that OpenShift users will likely be using `oc` and other OpenShift docs will reference `oc` so it's nice to keep that consistency.

Reply:
@jaypipes I'm looking through the CARM page and your notes here, trying to figure out a good way to simplify and dedupe the parts of our process that are the same as the existing one, but I'm running up against feeling like if I start to add little blurbs throughout the existing steps outlining the unique parts of the OpenShift process (ack-user-secrets, ack-user-config, the OLM install part, adding the namespace, the differences in using IAM users and roles instead of ServiceAccounts), that it will start to make the existing instructions really messy and confusing. Having them distinct the way it is now alleviates that complexity and confusion but is more verbose obviously and sometimes redundant. I'm not really sure what is the better approach. I've pinged @acornett21 and @komish for their input as well.

Reply:
I just pushed up some changes to the CARM doc that hopefully removes any repetition by linking to the previous steps when needed. I also removed the "Step" labeling for the OpenShift section to remove any confusion with the generic steps in the top, main body of the doc.

Reply:
If the goal is to reduce duplication, I feel like the latest updates solve for that. It might be confusing as it's not a top down approach, but it might be that's just my preference to consume documentation. If we get feedback saying the documentation is hard to understand, we can update in the future.